### ISSUE TITLE
Theme detection requires the post content in the posts list

### ISSUE THREAD

**Issue by sebastienros** [OpenLiveWriter/OpenLiveWriter#548]
It seems that the theme can only be detected if the post body is rendered as part as the blog's main page.

The theme I was targeting would only display the title in the list of posts, and I can see that OLW first issues a query on this main page once the temp post has been created. The log render:

```
OpenLiveWriter,1.10120,None,00088,09-Jan-2017 11:11:29.977,"Failed to download template Webpage: System.NullReferenceException: Object reference not set to an instance of an object.
   at System.Windows.Forms.Control.MarshaledInvoke(Control caller, Delegate method, Object[] args, Boolean synchronous)
   at System.Windows.Forms.Control.Invoke(Delegate method, Object[] args)
   at OpenLiveWriter.BlogClient.Detection.BlogEditingTemplateDetector.ParseWebpageIntoEditingTemplate_OnUIThread(Control uiContext, BlogPostRegionLocatorStrategy regionLocator, IProgressHost progress)
   at OpenLiveWriter.BlogClient.Detection.BlogEditingTemplateDetector.GetBlogTemplateFiles(IProgressHost progress, BlogPostRegionLocatorStrategy regionLocatorStrategy, BlogEditingTemplateStrategy[] templateStrategies, BlogEditingTemplateType[] templateTypes)",""
```

Then the theme detection stops and doesn't show any message.

Once I rendered the post body as part of the main blog page, theme detection worked successfully.

**Comment by nvella** [comment]
Just patched this whilst working on the static site gen support; OLW will attempt to navigate to the post in the event that it can't find the post content. I'll close this issue once there's a merged PR.

**Comment by nvella** [comment]
Fix included in #844.
